libraries.json

```diff
@@ -304,10 +304,15 @@
     "category": "Math",
     "maven": "https://repo1.maven.org/maven2/com/ionspin/kotlin/bignum/"
   },
+  {
+    "github": "Kodein-Framework/Kodein-DB",
+    "category": "Storage",
+    "maven": "https://dl.bintray.com/kodein-framework/Kodein-DB/org/kodein/db/kodein-db/"
```

Comment:
it's already added into `DataBase` catagory.
if you change category in already exist item - it will be better

Reply:
fixed, but renamed `Database` instead.
Sorry, didn't checked as on https://libs.kmp.icerock.dev there wasn't Kodein-DB near to SQL libraires.
Plus, category are not sorted in the dropdown so didn't saw it.